build.gradle

```diff
@@ -244,6 +299,11 @@ task generateJavadoc(type: Javadoc) {
     options.addStringOption('Xdoclint:none', '-quiet')
 }
 
+
+signing {
+    sign publishing.publications
+}
```

Comment:
How does signing work on github? Do we store the private key somewhere?

Reply:
yes, I have added a secret. I am planning write document on Sunday